src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncIteratorTests.cs

```diff
@@ -6499,13 +6500,21 @@ static async System.Collections.Generic.IAsyncEnumerable<int> Iter([EnumeratorCa
         yield return value++;
         await Task.Yield();
     }
+    static async Task Main()
+    {
+        await foreach (var i in Iter(42))
+        {
+            System.Console.Write(i);
+        }
+    }
 }";
-            var comp = CreateCompilationWithAsyncIterator(new[] { source, EnumeratorCancellationAttributeType });
+            var comp = CreateCompilationWithAsyncIterator(new[] { source, EnumeratorCancellationAttributeType }, TestOptions.DebugExe);
             comp.VerifyDiagnostics(
                 // (6,73): warning CS8424: The EnumeratorCancellationAttribute applied to parameter 'value' will have no effect. The attribute is only effective on a parameter of type CancellationToken in an async-enumerable method
                 //     static async System.Collections.Generic.IAsyncEnumerable<int> Iter([EnumeratorCancellation] int value)
                 Diagnostic(ErrorCode.WRN_UnconsumedEnumeratorCancellationAttributeUsage, "EnumeratorCancellation").WithArguments("value").WithLocation(6, 73)
                 );
+            CompileAndVerify(comp, expectedOutput: "42");
```

Comment:
Did this change affect the semantics of the test?

Reply:
Calling `CompileAndVerify` would have previously crashed.
Now it no longer crashes, and I added an execution test (because we can, but it is not essential).